celery/backends/redis.py

```diff
@@ -69,6 +69,8 @@
 
 E_LOST = 'Connection to Redis lost: Retry (%s/%s) %s.'
 
+E_CLIENT = "Error creating to Redis client: (%s)."
```

Comment:
Can you verify if this file is using for message broker backend or result store backend?

Reply:
This file is used for Redis result store backend..

Reply:
We have not enabled Result store backend. We are using Redis for message broker.